src/components/heading/heading.stories.mdx

```diff
@@ -53,7 +53,7 @@ You can control the appearance of a `c-heading` component by including a `c-head
 
 ## Weight Modifier
 
-There are some instances where the heading weight is lighter than the default `700`, this modifier will set the font weight to `300`.
+There are some instances where the heading weight is lighter than the default `font-weight.$bold`, this modifier will set the font weight to `font-weight.$light`.
```

Comment:
Two pieces of feedback:

- The part about `font-weight.$bold` is actually not 100% true, there are some heading levels where we use a heavier weight. You can just say something to the effect of the usual weight depends on the level.
- This comment says a lot about _what_, but not a lot about _why_. Why would someone opt for a lighter weight from a design standpoint? It has something to do with enforcing hierarchy without decreasing the prominence of a heading.